releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,72 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.1/1.2.1 - January 13, 2020
+
+* Added Support for Spring Boot 2.2.x Hoxton with version 1.2.1. Spring Boot 2.1.x Greenwich is still supported with 1.1.1.
+
+### Configuration
+
+* The property below has been removed as the timer-based configuration watch has been deprecated.
+
+```properties
+spring.cloud.azure.appconfiguration.watch.enabled
+```
+
+* The configuration is refreshed based on application activities.
+  * The configuration is cached and the default cache expiration time is 30 seconds. The configuration won't be refreshed until the cache expiration time window is reached. This value can be modified, see example below.
+  * In a web application, this library will signal configuration refresh automatically as long as there are incoming requests to the application. This is done by listening for ServletRequestHandledEvents.
+  * In a non-web application, the user needs to call AzureCloudConfigRefresh's `refreshConfigurations` method to signal configuration refresh at places where application activities occur. AzureCloudConfigRefresh can be accessed via dependency injection. Web based applications can also use this method, though both methods will still only update when the cache has expired. `refreshConfigurations` will return a `Future<Boolean>`, the resulting Boolean can be checked to see if a refresh event was triggered. **Note:** This does not mean the updated has been completed, only that a request was made to have the configurations updated.
```

Comment:
This does not mean the update~~d~~ has been completed...

If we are going to change the design of `AzureCloudConfigRefresh` soon, I am thinking we probably don't bother to advertise it now. Thoughts?

Reply:
I don't think any anything about this will change. The fail fast change don't happen until after `RefreshEvent` takes place. So if the failure happens when checking if a `RefreshEvent` needs to be called, then the user will be able to catch the error.

Reply:
I am talking about the name change of `AzureCloudConfigRefresh`.